### PR TITLE
return path from requested prefix instead of root when fetching namespace

### PR DESCRIPTION
**Pull request by pkasprzyk**
ref. https://neptune-labs.atlassian.net/browse/NPT-10417

before:
```
import neptune.new as neptunerun = neptune.init()
run['model/params'] = {'a': 3}
run.wait()
params run['model/params'].fetch() == {'model': {'params': {'a': 3}}}
```

after:
```
import neptune.new as neptunerun = neptune.init()
run['model/params'] = {'a': 3}
run.wait()
params run['model/params'].fetch() == {'a': 3}
```

Also refactored `fetch_atom_attribute_values` to make it more unit-testable.

### CODE REVIEW

File: neptune/new/attributes/namespace.py [hunk 1]
```diff
@@ -75,8 +74,14 @@ def _collect_atom_values(self, attribute_dict) -> dict:
...
     def fetch(self) -> dict:
-        namespace_values = self._backend.fetch_atom_attribute_values(self._run_uuid, self._path)
-        return self._collect_atom_values(namespace_values)
+        attributes = self._backend.fetch_atom_attribute_values(self._run_uuid, self._path)
+        from neptune.new.internal.run_structure import RunStructure
+        from neptune.new.internal.utils.paths import parse_path
+        run_struct = RunStructure()
+        prefix_len = len(self._path)
+        for attr_name, attr_type, attr_value in attributes:
+            run_struct.set(parse_path(attr_name)[prefix_len:], (attr_type, attr_value))
```
Comment (Raalsky) [inline neptune/new/attributes/namespace.py:83]: I think this "deserialization" logic should be placed in the RunStructure class.

File: neptune/new/attributes/namespace.py [hunk 2]
```diff
@@ -75,8 +74,14 @@ def _collect_atom_values(self, attribute_dict) -> dict:
         return result
 
     def fetch(self) -> dict:
-        namespace_values = self._backend.fetch_atom_attribute_values(self._run_uuid, self._path)
-        return self._collect_atom_values(namespace_values)
+        attributes = self._backend.fetch_atom_attribute_values(self._run_uuid, self._path)
+        from neptune.new.internal.run_structure import RunStructure
```
[the remark this answers is not in the document]
Comment (pkasprzyk) [reply neptune/new/attributes/namespace.py:78]: sorry, I did it while testing and forgot to clean it up :(

File: neptune/new/attributes/namespace.py (hunk 1 above)
Comment (pkasprzyk) [reply neptune/new/attributes/namespace.py:83]: do you mean `parse_path(attr_name)[prefix_len:]`?

the thing is I'm abusing `RunStructure` a little bit here (it's normally used for run **structure**, not values) so probably the by-the-book thing to do would to subclass `RunStructure` as `RunWithValuesStructure` and add deserialization there... not sure if it's worth it, though?

Comment (Raalsky) [reply neptune/new/attributes/namespace.py:82]: So maybe in some utils file/internal method in Namespace?

**Review by Raalsky — APPROVED** [review]

**Pull request merged**